packages/history/__tests__/hash-test.js

```diff
@@ -21,12 +21,19 @@ import BlockPopWithoutListening from './TestSequences/BlockPopWithoutListening.j
 // const canGoWithoutReload = window.navigator.userAgent.indexOf('Firefox') === -1;
 // const describeGo = canGoWithoutReload ? describe : describe.skip;
 
-describe('a hash history', () => {
+export const testHashHistory = (initialRoot, options) => {
```

Comment:
This test now exports a generic function to test both `createHashHistory()` and `createHashHistory({hashRoot: ""})`